dataclients/kubernetes/definitions.go

```diff
@@ -4,14 +4,11 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"sort"
 	"strconv"
 	"time"
-
-	log "github.com/sirupsen/logrus"
 )
 
-type resourceId struct {
+type resourceID struct {
```

Comment:
let's rename definitions to ingressdefinitions and routegroupcrd to routegroupdefinitions

Reply:
done